app/views/search-your-gp.nunjucks

```diff
@@ -25,10 +25,10 @@
         {% else %}
         <div class="form-group">
           {% endif %}
-          <label class="form-label-bold" for="location">{{ locationLabel }}</label>
+          <label class="form-label-bold" for="search">{{ searchLabel }}</label>
 
           <div class="form-group">
-            <input autocomplete="off" type="text" class="form-control" name="location" id="location" value=""{% if errorMessage %} aria-describedby="error-message" aria-invalid="true"{% endif %}>
+            <input autocomplete="off" type="text" class="form-control" name="search" id="search" value=""{% if errorMessage %} aria-describedby="error-message" aria-invalid="true"{% endif %}>
             <input type="hidden" name="context" value="{{ context }}">
```

Comment:
At the content review yesterday it was discussed that there is a need to show from where the user is coming from, so the context is still used to display the back to a specific condition/symptom a la pharmacy finder